sys/src/libsec/port/dh.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of the UCB release of Plan 9. It is subject to the license
+ * terms in the LICENSE file found in the top-level directory of this
+ * distribution and at http://akaros.cs.berkeley.edu/files/Plan9License. No
+ * part of the UCB release of Plan 9, including this file, may be copied,
+ * modified, propagated, or distributed except according to the terms contained
+ * in the LICENSE file.
+ */
+
+#include "os.h"
```

Comment:
What's os.h? Is it a .h that includes .h? Then it needs to not do that. .h do not include .h in our code base. I realize that rule is broken in places but ... let's try to get this right.